node/src/neighborhood/mod.rs

```diff
@@ -547,17 +547,22 @@ impl Neighborhood {
         }
     }
 
-    fn validate_database_min_hops_count(&self) {
+    fn validate_or_replace_min_hops_value(&mut self) {
         if let Some(persistent_config) = self.persistent_config_opt.as_ref() {
-            let from_db = persistent_config
-                .min_hops_count()
+            let value_in_db = persistent_config
+                .min_hops()
                 .expect("Min Hops Count value is not initialized inside Database");
-            let demanded = self.min_hops_count;
-            if demanded != from_db {
-                panic!(
-                    "Database with wrong min hops count value detected; expected: {:?}, was: {:?}",
-                    demanded, from_db
-                )
+            let value_in_neighborhood = self.min_hops;
+            if value_in_neighborhood != value_in_db {
+                debug!(
```

Comment:
I can imagine this should be `info!()`. Think, if this happen it means the user (although a bit unrealistic) managed to change the value in the meantime, then it makes sense to confirm for him that we detected the change and will use the new value, the same one as which was in his instructions.

Reply:
Done 👍